### ISSUE TITLE
Feature Request: Use TIME_SERIES_INTRADAY instead of TIME_SERIES_DAILY for non-currency quotes

### ISSUE THREAD

**Issue by urosevic** [urosevic/stock-ticker#1]
**Pros:**

- TIME_SERIES_INTRADAY contain full date/time of change while TIME_SERIES_DAILY mainly contain only date

**Cons:**

- TIME_SERIES_INTRADAY does not support currencies
- TIME_SERIES_INTRADAY  for indexes (like ^DJI) return zero for volume

Request source: https://wordpress.org/support/topic/support-for-intradaily/

**Comment by pstatho** [comment]
Any chance if this issue can get fixed faster if I donate?

**Comment by urosevic** [comment]
Here is a test version in branch _intraday_ - option is disabled by default, have to be enabled in plugin settings.

Intraday method is ignored for currencies (symbol ending with `=X`) and indexes because of missing volumes (symbols beginning with `^`).

https://github.com/urosevic/stock-ticker/tree/intraday

**Comment by pstatho** [comment]
Just wanted to let you know that Intraday seems to be working well for me so far. Thank you very much. I'll be sending you a donation as promised ;)

**Comment by urosevic** [comment]
Great to hear that it working well!
So, you think it's ready for public release? Have you noticed any issue during testing (at least delay in retrieving quotes or stuck refresh (like quote not refreshed for couple days)?

And, thank you for your support!

**Comment by pstatho** [comment]
The random times I've checked, it was being updated every half-hour but that's likely due to a limitation of AlphaVantage that will only update every 1/2 hour. For me that's perfect.

I would say it's good to release publicly.

**Comment by urosevic** [comment]
Thanks, just released v3.0.5 with some more fixes and documentation improvements.

Btw, I found that RANGE and VOLUME are not good for intraday time series, because they are composed for selected timeframe (15 minutes in v3.0.5).

Btw, test version was with 30min timeframe, but I changed for v3.0.5 to 15min (1/2 hour update frequency you mentioned).

If you notice any issue with this shorter timeframe, feel free to report here.
